backend/mora/service/orgunit.py

```diff
@@ -42,6 +43,42 @@
                             url_prefix='/service')
 
 
+def _read_local_settings(unitid=None):
```

Comment:
When the `get` functions in `configuration_options.py` has been split out, the call to this function can essentially be replaced by that - and this function be completely removed.